packages/mui-system/src/index.js

```diff
@@ -31,6 +33,14 @@ export {
   extendSxProp as unstable_extendSxProp,
   unstable_defaultSxConfig,
 } from './styleFunctionSx';
+// TODO: Remove this function in v6
+// eslint-disable-next-line @typescript-eslint/naming-convention
+export function experimental_sx() {
+  throw new MuiError(
+    'MUI: The `experimental_sx` has been moved to `theme.unstable_sx`.' +
+      'You should remove the import and replace the usage with `theme.unstable_sx(…same value you pass to experimental_sx)` instead.',
```

Comment:
Is there a specific reason why we don't use the same error message?

Reply:
Alright fair enough. Should we keep ""MUI: The `experimental_sx` has been moved to `theme.unstable_sx`.For more details, see https://github.com/mui/material-ui/pull/35150" instead of the current value. There are three new error messages in error-codes.js

Reply:
fine for me.